### PR TITLE
Test cases on Sidebar position for Pages & posts

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Description

Written Test Cases on Sidebar position [Default Layout for Sidebar] for Pages & posts
### How has this been tested?

For Post :-  npm run test:e2e:interactive — tests/e2e/specs/customizer/sidebar/post-sidebar-test.js
For Page :- npm run test:e2e:interactive — tests/e2e/specs/customizer/sidebar/pages-sidebar-test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/sidebar/pages-sidebar-test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            'single-page-sidebar-layout': 'left-sidebar',
+
+        };
+        await setCustomize( Pagesidebar );
+
+
+
+		await page.goto( createURL( 'sample-page' ), {
+			waitUntil: 'networkidle0',
+		} );
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/pages-sidebar-test.js:15]: waitforselector functions are missing

File: tests/e2e/specs/customizer/sidebar/pages-sidebar-test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        await expect( {
+            selector: '.ast-separate-container.ast-right-sidebar #secondary, .ast-separate-container.ast-left-sidebar #secondary',
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/pages-sidebar-test.js:17]: More than one selector passed here - Is this running at your end? Not working for me though

File: tests/e2e/specs/customizer/sidebar/post-sidebar-test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { setCustomize } from "../../../utils/set-customize";
+import { createURL,createNewPost,publishPost } from "@wordpress/e2e-test-utils";
+describe('Sidebar for posts', () => {
+    it( 'Position for Sidebar for blog-posts  should apply corectly', async () => {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/sidebar/post-sidebar-test.js:4]: Add single spaces between 2 words. I can see  two spaces between these 2 words "blog-posts should" in every testcase

File: tests/e2e/specs/customizer/sidebar/pages-sidebar-test.js (hunk 2 above)
Comment (bsfsnehal) [reply tests/e2e/specs/customizer/sidebar/pages-sidebar-test.js:18]: Okay I will remove it

Comment (bsfsnehal) [reply tests/e2e/specs/customizer/sidebar/pages-sidebar-test.js:18]: yes, it is running

**Pull request closed without merging**